plugins.d/python.d.plugin

```diff
@@ -7,21 +7,79 @@ import os
 import sys
 import time
 
+
+MODULE_EXTENSION = ".chart.py"
+BASE_CONFIG = {'update_every': 1,
+               'priority': 90000,
+               'retries': 10}
+
+
+# -----------------------------------------------------------------------------
+# logging functions
+
+PROGRAM = os.path.basename(__file__).replace(".plugin", "")
+DEBUG_FLAG = False
+
+def debug(*args):
+    """
+    Print message on stderr.
+    """
+    if not DEBUG_FLAG:
+        return
+    sys.stderr.write(PROGRAM + " DEBUG :")
+    for i in args:
+        sys.stderr.write(" " + str(i))
+    sys.stderr.write("\n")
+    sys.stderr.flush()
+
+def error(*args):
+    """
+    Print message on stderr.
+    """
+    sys.stderr.write(PROGRAM + " ERROR :")
+    for i in args:
+        sys.stderr.write(" " + str(i))
+    sys.stderr.write("\n")
+    sys.stderr.flush()
+
+def info(*args):
+    """
+    Print message on stderr.
+    """
+    sys.stderr.write(PROGRAM + " INFO :")
+    for i in args:
+        sys.stderr.write(" " + str(i))
+    sys.stderr.write("\n")
+    sys.stderr.flush()
+
+def fatal(*args):
+    """
+    Print message on stderr and exit.
+    """
+    sys.stderr.write(PROGRAM + " FATAL :")
+    for i in args:
+        sys.stderr.write(" " + str(i))
+    sys.stderr.write("\n")
+    sys.stderr.flush()
+    sys.stdout.write('DISABLE\n')
+    sys.exit(1)
+
+
+
+# -----------------------------------------------------------------------------
+# globals & python modules management
+
 # setup environment
 # https://github.com/firehol/netdata/wiki/External-Plugins#environment-variables
-MODULES_DIR = os.getenv('NETDATA_PLUGINS_DIR',
-                        os.path.abspath(__file__).strip("python.d.plugin.py").replace("plugins.d", ""))
+MODULES_DIR = os.path.abspath(os.getenv('NETDATA_PLUGINS_DIR',
+                        os.path.dirname(__file__)) + "/../python.d") + "/"
 CONFIG_DIR = os.getenv('NETDATA_CONFIG_DIR', "/etc/netdata/")
-INTERVAL = os.getenv('NETDATA_UPDATE_EVERY', None)
+UPDATE_EVERY = os.getenv('NETDATA_UPDATE_EVERY', 1)
```

Comment:
Ok, let me state how it should work (like the rest of netdata):

1. The environment variable `NETDATA_UPDATE_EVERY` is the default update_every for every plugin (it should default to 1, if unset)
2. A numeric command line argument overwrites this global default
3. The option `update_every` in `python.d.conf` overwrites this global default
4. The option `update_every` in `python.d/MODULE.chart.py` sets the default for all jobs of a module (if unset the global default will be set as module default)
5. The option `update_every` in a job in `python.d/MODULE.chart.py` sets the update frequency of the specific job. If unset, the module default will be used as the job update_every.

I am not sure how to fix it. Do you want me to cancel this PR and use it as a base to fix it?